library/general/src/lib/ui/greasemonkey.rb

```diff
@@ -0,0 +1,225 @@
+# encoding: utf-8
+
+# Copyright (c) 2012 Novell, Inc.
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may
+# find current contact information at www.novell.com.
+
+# File:	Greasemonkey.ycp
+# Package:	yast2-storage
+# Summary:	Expert Partitioner
+# Authors:	Arvin Schnell <aschnell@suse.de>
+#
+# Lets see if this turns out to be useful.
+require "yast"
+
+module UI
+  # UI layout helpers.
+  #
+  module Greasemonkey
```

Comment:
if you want to add GreaseMonkey to yast2, then significant improvement of code quality is needed ;) as yast2 mean that it will be used on more places and there is highed requirements. Also documentation have to be significantly improved and same for test coverage.